public/css/header.module.css

```diff
@@ -1,47 +1,55 @@
 .header {
-    color: #fff;
-    background-color: rgba(255, 105, 105, 0.6);
-    padding: 0;
-    margin: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: #fff;
+  background-color: rgba(0, 0, 0, 0.5);
+  width: 100%;
+  height: 10%;
+  position: absolute;
+  user-select: none;
+  & > * {
```

Comment:
add an empty line between these lines to better separate the header's properties and the other selector